### PR TITLE
Components calculator UI

### PR DESCRIPTION
**Pull request by MattGomb**
In this PR I created the calculator.js file and styled it, according to the requirements.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,17 +1,28 @@
-import React from "react";
+import React from 'react';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-  }
+const Calculator = () => (
```
Comment (MahmoudBakr23) [inline src/components/Calculator.js]: - [ ] In this milestone, the component should be class-based, please change it back 😄

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @MattGomb 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done creating your Calculator component so far
- Nice design
- Descriptive PR and README file
- Correct Linters

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, not criticizing it. 😄

 ## Required Changes ♻️

- [ ] Please create the root component `<App />` and import the `<Calculator />` component in it, then use it finally in the index file. Note that the `App` component should also be class-based component.

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (MattGomb) [reply src/components/Calculator.js]: @MahmoudBakr23 Hi, in this case, can you tell me how to bypass this error:  
`Line 3:1:  Component should be written as a pure function  react/prefer-stateless-function`
?
Thanks!

Comment (MattGomb) [reply src/components/Calculator.js]: @MahmoudBakr23 Because this way it does not compile

Comment (MattGomb) [reply src/components/Calculator.js]: @MahmoudBakr23  
Can you please tell me what to do with this?
image

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

## Hey @MattGomb :wave:

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Following GitFlow
:heavy_check_mark: Following Requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**